### PR TITLE
Refactor to componentized/built React app.

### PR DESCRIPTION
**Pull request by jpetto**
- Move from gh-pages branch to /docs folder on master.

### CODE REVIEW

**Comment by jpetto** [comment]
The code in this PR is _almost_ identical to #12, though a small change was made to `/local-dev/scripts/finalize.js` to silently fail if/when trying to move `/local-dev/build/status.yml` to `/docs` (as `/local-dev/build/status.yml` would only exist for testing purposes and should not overwrite the canonical version in `/docs`). Noting so someone can take another peek at `finalize.js` to make sure my coffee worked this morning.

If/when this PR merges, we'll need to delete the `gh-pages` branch, update the Github Pages config to point to the `/docs` folder, and update any scripts that make changes to `status.yml` (as it now resides in the `/docs` folder of the **master** branch).

File: local-dev/scripts/finalize.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        fs.readdir(deployPath, (err, items) => {
+            // iterate over each item in deployPath
+            for (let item of items) {
+                // get stats on each item
+                fs.stat(deployPath + '/' + item, (err, stats) => {
+                    // if no error and item is a file and file is *not* blessed, remove it
+                    if (!err && stats && stats.isFile() && blessedFiles.indexOf(item) === -1) {
+                        fs.remove(deployPath + '/' + item, err => {
+                            if (!err) {
+                                console.log('🗑 Removed ' + item + '!');
```
Comment (glogiotatidis) [inline local-dev/scripts/finalize.js:25]: Fancy icons in my terminal. Maybe a bit too hip for my taste but I'll let you have it :)

**Comment by glogiotatidis** [comment]
Good job @jpetto 🍰 !

I'll work on the required worker updates before merging this.

**Comment by glogiotatidis** [comment]
Created develop branch and mozmar/status.ramzom.org, to act as our staging site. 

I merged this one in `develop` (https://github.com/mozmar/status.mozmar.org/commit/4442b2290d3ceaf903279d16f727bfbb940fc76b) which means its not live yet. I'll have to continue the refactor tomorrow with the python worker. 

The plan is to PR against `develop` and merge to `master` when a new release is ready. With this I'm closing this PR. Thanks John!

**Comment by glogiotatidis** [comment]
This is now in production! 

Please send new PRs against `develop` branch which is served from `https://status.ramzom.org`. When done we merge to `master` to update production.

**Pull request closed without merging**